src/igvfd/types/file_set.py

```diff
@@ -632,6 +632,9 @@ class ConstructLibrarySet(FileSet):
         Path('associated_phenotypes', include=['@id', 'term_id', 'term_name']),
         Path('small_scale_gene_list', include=['@id', 'geneid', 'symbol', 'name', 'synonyms']),
         Path('applied_to_samples', include=['@id', 'accession', 'aliases']),
+        Path('large_scale_gene_list', include=['@id', 'accession', 'aliases']),
+        Path('large_scale_loci_list', include=['@id', 'accession', 'aliases']),
```

Comment:
Could you also add this to prediction sets while you are at if you don't mind?

Reply:
yeah!